<launch>
  <arg name="speech_topic" default="/speech"/>

  <node name="hand_caffe" pkg="aero_enshu"
        type="hand_caffe" output="screen">
     <remap from="/windows/voice" to="$(arg speech_topic)"/>
  </node>

  <node name="simulated_speak" pkg="negomo_enshu"
        type="simulated_speak.py" output="screen"/>

  <node name="example_interaction_cb" pkg="aero_enshu"
        type="interaction.py" output="screen">
     <param name="taskto" value="task2"/>
     <param name="phrase" value="I will hand over this item in a moment."/>
     <param name="time" value="1.0"/>
     <param name="phrase_refuse" value="I cannot interact at the moment."/>
     <param name="time_refuse" value="1.0"/>
  </node>

  <node name="negomo_sensor_hand" pkg="aero_enshu"
        type="detect_hand" output="screen"/>

  <node name="negomo" pkg="negomo_enshu"
        type="negotiation_model">
     <rosparam file="$(find negomo_enshu)/config/negomo.yaml"
               command="load"/>
  </node>

  <include file="$(find negomo_enshu)/examples/launch/plot.launch"/>
</launch>